site2/docs/client-libraries-go.md

```diff
@@ -846,43 +868,43 @@ ctx := context.Background()
 
 // create producer
 producer, err := client.CreateProducer(pulsar.ProducerOptions{
-	Topic:           topic,
-	DisableBatching: true,
+    Topic:           topic,
+    DisableBatching: true,
 })
 if err != nil {
-	log.Fatal(err)
+    log.Fatal(err)
 }
 defer producer.Close()
 
 // send 10 messages
-msgIDs := [10]MessageID{}
+msgIDs := [10]pulsar.MessageID{}
 for i := 0; i < 10; i++ {
-	msgID, err := producer.Send(ctx, &pulsar.ProducerMessage{
-		Payload: []byte(fmt.Sprintf("hello-%d", i)),
-	})
-	assert.NoError(t, err)
-	assert.NotNil(t, msgID)
-	msgIDs[i] = msgID
+    msgID, _ := producer.Send(ctx, &pulsar.ProducerMessage{
```

Comment:
@nodece 
could you please review this PR from a technical perspective? Thank you!

Reply:
Seems that the original code snippet is copied from the test cases, which is unnecessary in the documentation.

Reply:
Hi @Anonymitaet, I suggest we remove parameter details, and then use https://pkg.go.dev/github.com/apache/pulsar-client-go/pulsar instead, when the go-client updates, we need to update our documentation again, what do you think of this.

Reply:
I suggest reserving them because other clients' documentation also has parameter details.

Reply:
+1 for removing them. It's hard to maintain the parameters detail in two places: the go doc and the pulsar doc website. Or we can provide some commonly used parameters in this doc and let's user jump to go doc for more detail.
In fact, the java client doc also encourages the user to jump to the java doc to get more details. But leave the loadConf configuration which is generated from the code in the doc.

Reply:
@nodece I'm on your side. Referencing the source material reduces maintenance cost

@labuladong FYI: to reduce manual work, we (@SignorMercurio @urfreespace) are trying to generate client configurations docs from code automatically (https://github.com/apache/pulsar/pull/17198), which means we'll remove the [configuration instructions - table](https://pulsar.apache.org/docs/next/client-libraries-java#configure-producer) from client doc.

Reply:
Got it. I will remove the parameter details and link to https://pkg.go.dev/github.com/apache/pulsar-client-go/pulsar